content/v2.0/get-started.md

```diff
@@ -194,6 +195,100 @@ docker exec -it influxdb /bin/bash
 {{% /tab-content %}}
 <!--------------------------------- END Docker -------------------------------->
 
+<!-------------------------------- BEGIN kubernetes---------------------------->
+{{% tab-content %}}
+
+You can install InfluxDB in a local kubernetes environment with Minikube.
+
+1. Install minikube.
+
+    On macOS, run:
```

Comment:
Rather than make this macOS specific, just point them to the Minikube installation docs: https://kubernetes.io/docs/tasks/tools/install-minikube/

These provided instructions for multiple OSs.